frontend/public/components/nav.jsx

```diff
@@ -49,9 +49,8 @@ class NavLink extends React.PureComponent {
 class ResourceNSLink extends NavLink {
   static isActive (props, resourcePath, activeNamespace) {
     const href = stripNS(formatNamespacedRouteForResource(props.resource, activeNamespace));
-    return props.model
-      ? resourcePath === referenceForModel(props.model) || _.startsWith(resourcePath, `${referenceForModel(props.model)}/`)
-      : resourcePath === href || _.startsWith(resourcePath, `${href}/`);
+    return (props.model && (resourcePath === referenceForModel(props.model) || _.startsWith(resourcePath, `${referenceForModel(props.model)}/`)))
+      || resourcePath === href || _.startsWith(resourcePath, `${href}/`);
```

Comment:
Logic looks correct to me, although it's complicated enough now that it might be worth breaking it into multiple methods. Something like... (untested)

```js
static matchesPath(actual, prefix) {
  return actual === prefix || _.startsWith(actual, `${prefix}/`);
}

static matchesModel(resourcePath, model) {
  return model && ResourceNSLink.matchesPath(resourcePath, referenceForModel(model));
}

// ...
return ResourceNSLink.matchesPath(resourcePath, href) || ResourceNSLink.matchesModel(resourcePath, props.model);
```

Reply:
True, and I may prefer them as standalone functions rather than tack them onto the `Component`.

Reply:
@spadgett Good idea, made them into separate helper functions.